### ISSUE TITLE
Panic: no Auth Provider found for name "gcp"

### ISSUE THREAD

**Issue by 3j0s** [kubernetes/client-go#839]
Greetings!

I'm using the configuration as follow in this Pastebin:
[Panic: no Auth Provider found for name "gcp"](https://pastebin.com/hykBMerf)

This is the Pastebin go.mod: [go.mod](https://pastebin.com/gTPN0sQ3)

I'm running:

- GKE Cloud: 1.17.9-gke.600 

- GO:  go1.14.6 linux/amd64

-  Kubectl: Major:"1", Minor:"18", GitVersion:"v1.18.3"

-  Ubuntu: 5.3.0-1032-gcp #34~18.04.1-Ubuntu SMP

This is the error that I got:

`Log file created at: 2020/08/05 05:49:49
Running on machine: gke-admin
Binary: Built with gc go1.14.6 for linux/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
F0805 05:49:49.453823   20353 k8s-deploy.go:62] Could not read kubeconfig /home/infraops/.kube/config: no Auth Provider found for name "gcp"`

Can anybody help me with this please?, thx in advance

**Comment by liggitt** [comment]
have you enabled the auth plugin? see https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go#L33-L40 for examples of enabling all auth plugins or just the gcp one

**Comment by 3j0s** [comment]
Regards!

Thx for your response, when I enabled the plugin it threw me the following
error:
~] ./k8s-deploy.go:37:9: imported and not used: "
k8s.io/client-go/plugin/pkg/client/auth/gcp"

But, 'cause you ask me about it, I uncommented the line again and I solve
the problem with this alias " _ ", so the import plugin stayed like this:

    _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"

Now, it's working, sorry for my lack of experience.
Thank you again.
